app/src/organisms/CalibrationPanels/useConfirmCrashRecovery.tsx

```diff
@@ -21,18 +19,17 @@ import type { CalibrationPanelProps } from './types'
 const CONDITION = 'Jog too far or bend a tip?'
 const START_OVER = 'Start over'
 
-export type Props = {|
-  requiresNewTip: boolean,
-  ...CalibrationPanelProps,
-|}
+export interface Props extends CalibrationPanelProps {
+  requiresNewTip: boolean
+}
 
 export function useConfirmCrashRecovery(
   props: Props
-): [React.Node, React.Node] {
+): [React.ReactNode, React.ReactNode] {
```

Comment:
Nice opportunity for a TypeScript [labeled tuple](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-4-0.html#labeled-tuple-elements) here

```suggestion
): [message: React.ReactNode, modal: React.ReactNode] {
```